package.json

```diff
@@ -9,6 +9,10 @@
   "author": "Deta",
   "license": "MIT",
   "dependencies": {
+    "eslint-config-airbnb": "^18.2.1",
```

Comment:
these should only be dev dependencies, also we don't require
`jsx` and `react` plugins.